### PR TITLE
stolostron/backlog#19974 Upd clst pools img 2.4

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/cluster_pool_rel_img_update.adoc [hunk 1]
```diff
@@ -7,7 +7,10 @@ When the clusters in your cluster pool remain in hibernation for some time, the
 
 Complete the following steps to update the {ocp-short} release image for the clusters in your cluster pool:
 
-*Note:* This procedure does not update clusters from the cluster pool that are already claimed or hibernating in the cluster pool. After you complete this procedure. only clusters that are created and added to the cluster pool use the updated release image. 
+*Note:* This procedure does not update clusters from the cluster pool that are already claimed in the cluster pool. After you complete this procedure, the release images are only updated to the following clusters related to the cluster pool:
+* Clusters that are created by the cluster pool after updating the release image with this procedure. 
+* Clusters that are added to the cluster pool updating the releaes image with this procedure.
```
Comment (KevinFCormier) [inline clusters/cluster_pool_rel_img_update.adoc]: I don't think this line is necessary. You can't add clusters to a cluster pool. The cluster pool creates all clusters in the pool.

File: clusters/cluster_pool_rel_img_update.adoc [hunk 2]
```diff
@@ -7,7 +7,10 @@ When the clusters in your cluster pool remain in hibernation for some time, the
 
 Complete the following steps to update the {ocp-short} release image for the clusters in your cluster pool:
 
-*Note:* This procedure does not update clusters from the cluster pool that are already claimed or hibernating in the cluster pool. After you complete this procedure. only clusters that are created and added to the cluster pool use the updated release image. 
+*Note:* This procedure does not update clusters from the cluster pool that are already claimed in the cluster pool. After you complete this procedure, the release images are only updated to the following clusters related to the cluster pool:
```
Comment (dtthuynh) [inline clusters/cluster_pool_rel_img_update.adoc]: Does this flow a bit better?
`After you complete this procedure, the release image update only applies to the following clusters related to the cluster pool:`

File: clusters/cluster_pool_rel_img_update.adoc (hunk 1 above)
Comment (cadawson) [reply clusters/cluster_pool_rel_img_update.adoc]: Removed the line.

File: clusters/cluster_pool_rel_img_update.adoc (hunk 2 above)
Comment (cadawson) [reply clusters/cluster_pool_rel_img_update.adoc]: Modified the text a bit.

**Review by KevinFCormier — APPROVED** [review]
Looks good!

**Review by dtthuynh — APPROVED** [review]

**Pull request merged**